src/ast/semantic_analyser.cpp

```diff
@@ -1296,17 +1296,17 @@ bool SemanticAnalyser::check_arg(const Call &call, Type type, int arg_num, bool
   return true;
 }
 
-bool SemanticAnalyser::check_alpha_numeric(const Call &call, int arg_num __attribute__((unused)))
+bool SemanticAnalyser::check_symbol(const Call &call, int arg_num __attribute__((unused)))
 {
   if (!call.vargs)
     return false;
 
   auto &arg = static_cast<String&>(*call.vargs->at(0)).str;
 
-  bool is_alpha = std::regex_match(arg, std::regex("^[a-zA-Z0-9_-]+$"));
+  bool is_alpha = std::regex_match(arg, std::regex("^[a-zA-Z0-9./_-]+$"));
```

Comment:
Maybe change the variable name too
```suggestion
  bool is_valid = std::regex_match(arg, std::regex("^[a-zA-Z0-9./_-]+$"));
```

Reply:
Good catch